examples/hello-world/format-example.cpp

```diff
@@ -0,0 +1,144 @@
+
+#include "internalGroup1.h"
+#include "slang.h"
+#include <algorithm>
+
+#include "includeGroup2.h"
+#include <vector>
+
+#include <vulkan.h>
+
+#ifdef __linux__
+#    include <X11/Xlib.h>
+#else
+#    include <Windows.h>
+#endif
+
+namespace slang
+{
+/// <summary>
+/// Summary of TestClass
+/// </summary>
+class TestClass
+{
+private:
+    int examplePrivateMember; // Trailing comment1
+    int* examplePointerMember = nullptr; // trailing coment2
```

Comment:
Is there a parameter to keep these comments lined up?

Reply:
There is. But I preferred not to use them because they cause more than necessary changes and lead to overhead in code reviews. If you are strongly against this style we can change.

Reply:
My two thoughts on this:

* I would object strongly if the automated formatting *stops* me from aligning comments/names/etc. across multiple declarations. If I can manually align things and have the automatic formatting not mess that up, I'm okay with leaving its default behavior as not enforcing alignment.

* The agree that having alignment create extra diffs is a bit worrying. For code reviews you can use options to ignore whitespace (GitHub's diff viewer includes that as an option), but for things like merges the result could be more painful.

One potential saving grace for this kind of automated alignment is that it should only lead to whitespace changes when you modify at least one declaration in the block, which hopefully wouldn't collide with another change unless it *also* modified a declaration in the same block. The main case where that could lead to merge conflicts is any kind of bulk renaming of types, which is the kind of thing that can easily create conflicts anyway.